From 420425b7b99f819199d4b21acdde3450b8aac3a4 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Sun, 27 Aug 2023 09:19:24 +0000
Subject: [PATCH] Disabling nnhal for now as aidl version is required

aidl version of nnhal is required with API 34
and target 8 on android u. Disabling it till aidl
version is upgraded.

Tracked-On: OAM-110604_
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 caas/mixins.spec | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/caas/mixins.spec b/caas/mixins.spec
index 284a2d15..4ed1c721 100755
--- a/caas/mixins.spec
+++ b/caas/mixins.spec
@@ -89,4 +89,5 @@ sensors: mediation(enable_sensor_list=true)
 bugreport: true
 mainline-mod: true
 houdini: true
-neuralnetworks: true(vsock-remote-infer=true)
+#neuralnetworks: true(vsock-remote-infer=true)
+neuralnetworks: false
-- 
2.39.2

